src/sxapi/cli/example-config.conf

```
@@ -0,0 +1,16 @@
+[SXAPI]
+
+# smaXtec user name
+USER = smaxtec@user.com
```

please use example.com as domain for this:

```suggestion
USER = user.name@example.com
```